apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -187,6 +188,16 @@ export const UnconnectedTeacherHomepage = ({
             solidBorder={true}
           />
         )}
+        {showReturnToReopenedTeacherApplication && (
+          <BorderedCallToAction
+            headingText="Return to Your Application"
+            descriptionText="Your Regional Partner has requested updates to your Professional Learning Application."
+            buttonText="Return to Application"
+            buttonColor={Button.ButtonColor.orange}
+            buttonUrl="/pd/application/teacher?enableExperiments=teacher-application-saving-reopening"
```

Comment:
maybe add a TODO comment here like you've been doing elsewhere to remove the experiment stuff after launch

Reply:
It's up at the top!

Reply:
oh sorry i missed that! perfect!